src/igvfd/tests/test_metadata_metadata.py

```diff
@@ -691,6 +806,119 @@ def test_metadata_metadata_report_split_column_and_fields_by_experiment_and_file
         assert tuple(expected_experiment_column_to_fields_mapping[k]) == tuple(v), f'{k, v} not in expected'
 
 
+def test_metadata_metadata_report_v2_split_column_and_fields_by_experiment_and_file(dummy_request):
+    from igvfd.metadata.metadata import MetadataReportV2
+    dummy_request.environ['QUERY_STRING'] = (
+        'type=FileSet'
+    )
+    mr = MetadataReportV2(dummy_request)
+    mr._split_column_and_fields_by_experiment_and_file()
+    expected_file_column_to_fields_mapping = dict(
+        [
+            ('File ID', ['@id']),
+            ('File download URL', ['href']),
+            ('File accession', ['accession']),
+            ('File format', ['file_format']),
+            ('File format type', ['file_format_type']),
+            ('File content type', ['content_type']),
+            ('File summary', ['summary']),
+            ('Cell type annotation', ['cell_type_annotation.term_name']),
+            ('Creation timestamp', ['creation_timestamp']),
+            ('File size', ['file_size']),
+            ('File S3 URI', ['s3_uri']),
+            ('File assembly', ['assembly']),
+            ('File transcriptome annotation', ['transcritome_annotation']),
```

Comment:
typo also here, so not catching the above typo.

Reply:
Nice, fixed.